api/base/serializers.py

```diff
@@ -272,6 +291,7 @@ def get_url(self, obj, view_name, request, format):
 
 
 class JSONAPIListSerializer(ser.ListSerializer):
+    DEFAULT_BULK_LIMIT = 10
```

Comment:
This setting is used in contexts outside of this class, which argues for it being a global variable or shared setting (independent of this class).

See example usage: https://github.com/CenterForOpenScience/osf.io/pull/4353/files#diff-51eef6ea2a6c35e2b080c9a6fe591e38R397